pycobertura/reporters.py

```diff
@@ -234,6 +244,38 @@ def generate(self):
         return tabulate(lines, headers=headers)
 
 
+class CsvReporterDelta(DeltaReporter):
+    not_available = ""
+
+    def generate(self, delimiter):
+        lines = self.get_report_lines()
+
+        list_of_lines = [headers_without_missing]
+        list_of_lines.extend(
+            [[f"{item}" for item in row[:-1]] for row in zip(*lines.values())]
+        )
+
+        if self.show_source:
+            list_of_lines = [headers_with_missing]
+            list_of_lines.extend(
+                [[f"{item}" for item in row[:-1]] for row in zip(*lines.values())]
+            )
+            for count in range(1, len(list_of_lines)):
+                missing_line = lines["Missing"][count - 1]
+                list_of_lines[count] += [
+                    f"{[self.color_number(number) for number in missing_line]}".encode(
+                        "utf-8"
+                    ).decode("unicode_escape")
+                ]
+
+        if "\\n" in repr(delimiter):
```

Comment:
Why do we use `repr()` here for string equality? What is the output of `delimiter` without `repr()`?

Reply:
addressed here: https://github.com/aconrad/pycobertura/pull/130/commits/7c7635fc525d5d531a4826b415bda76d8bba0eac.

It has to do with the fact that backslashes are escaped with another backslash in strings and ultimately we would not get the right representation. Now I have found another way to do this :)